tests/test_unit_conversion.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+import os
+import cellmlmanip
+
+OXMETA = "https://chaste.comlab.ox.ac.uk/cellml/ns/oxford-metadata"
+
+
+@pytest.fixture
+def model():
+    return cellmlmanip.load_model(os.path.join(os.path.dirname(__file__), 'cellml_files', "test_simple_odes.cellml"))
+
+
+def test_add_preferred_custom_unit_name(model):
+    time_var = model.get_symbol_by_ontology_term(OXMETA, "time")
+    assert str(model.units.summarise_units(time_var)) == "ms"
+    model.units.add_preferred_custom_unit_name('millisecond', [{'prefix': 'milli', 'units': 'second'}])
+    assert str(model.units.summarise_units(time_var)) == "millisecond"
+    model.units.add_preferred_custom_unit_name('millisecond', [{'prefix': 'milli', 'units': 'second'}])
```

Comment:
Why doing the same thing twice?

Reply:
because the code does something slightly different if the unit already exists. In short it is to guarantee 100% code coverage of the diff.

Reply:
I'd add a comment to that effect then.

A stronger test case might be where the unit exists through another pathway than this new method?

Reply:
I don't think there is any other way to trigger the except within `add_preferred_custom_unit_name` other than adding the same unit twice

Reply:
comment added